meson.build

```
@@ -0,0 +1,544 @@
+project('vips', 'c', 'cpp',
+    version: '8.13.0',
+    meson_version: '>=0.56',
+    default_options: [
+        'c_std=c99'
+    ]
+)
+
+version_parts = meson.project_version().split('.')
+version_major = version_parts[0]
+version_minor = version_parts[1]
+version_patch = version_parts[2]
+
+# rules:
+# sources changed: increment revision
+# binary interface changed: increment current, reset revision to 0
+#   binary interface changes backwards compatible?: increment age
+#   binary interface changes not backwards compatible?: reset age to 0
+library_current = 57
+library_age = 15
+library_revision = 0
+library_version = '@0@.@1@.@2@'.format(library_current - library_age, library_age, library_revision)
+
+gnome = import('gnome')
+pymod = import('python')
+pkg = import('pkgconfig')
+i18n = import('i18n')
+
+add_project_arguments('-Drestrict=__restrict', language: 'cpp')
+add_project_link_arguments(
+    '-no-undefined',
+    language: 'c',
+)
+
+cc = meson.get_compiler('c')
+cpp = meson.get_compiler('cpp')
+
+glib_dep = dependency('glib-2.0')
```

Note to myself: minimum required GLib version is missing here.
```suggestion
glib_dep = dependency('glib-2.0', version: '>=2.40')
```